### ISSUE TITLE
Write more lab vars into REDCap

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#51]
When we created the REDCap form `` in the Results project, we left out a lot of variables that I thought didn't matter.  I was wrong about many of those. We need to add them so we can use them for data alignment and output. 

`get_lab_result_data` should include these columns in `col_types`

```
    `Home Phone` = col_character(),
    `Patient Cell Phone` = col_character(),
    `Work Phone` = col_character()
```

The new select statement in `get_lab_result_data` should probably read:

```
order_id,
order_date,
collection_date = collected,
result_date = lab_order_result_datetime,
mrn,
patient,
result_component,
acc_number,
results,
uf_id_from_order_question = uf_id_from_order_q,
uf_id_from_epic = uf_id,
age,
sex,
race,
zip,
county,
patient_email_address,
home_phone,
patient_cell_phone,
work_phone
```

That will require these variables be added to the REDCap Results project:

```
order_id
...
age
sex
race
zip
county
patient_email_address
home_phone
patient_cell_phone
work_phone
```

Note that `age` is expressed like "34-year old" in the Epic lab results dataset. Also, in subsequent reloads of the lab data it changes. It appears to be the patient age at report generation, not at sample collection. We will need to discuss what to do with `age`.

Other changes not yet described here might be required to add these vars.

**Comment by pbchase** [comment]
@tlstoffs, we will need your help with this request as we will need to add REDcap fields for storing the additional lab data.

**Comment by tlstoffs** [comment]
does the "..." mean everything? Or just 
order_id 
age
sex
race
zip
county
patient_email_address
home_phone
patient_cell_phone
work_phone

**Comment by pbchase** [comment]
Sorry, that was confusing.  I should have said...

These variables need to be added:

```
order_id
age
sex
race
zip
county
patient_email_address
home_phone
patient_cell_phone
work_phone
```

I suggest this order

```
order_id
<All existing vars in the redcap project>
age
sex
race
zip
county
patient_email_address
home_phone
patient_cell_phone
work_phone
```

**Comment by pbchase** [comment]
@tlstoffs do you have what you need to do this one?

**Comment by tlstoffs** [comment]
done

**Comment by pbchase** [comment]
The product of this PR will expand the amount of data available to Issues #68

**Comment by pbchase** [comment]
addressed by PR #58
